lua/compe_tabnine/init.lua

```diff
@@ -61,10 +61,14 @@ end
 --- determine
 function Source.determine(_, context)
 	-- dump(context)
-	return compe.helper.determine(context)
+	return compe.helper.determine(context, {
+			trigger_characters = vim.split(
```

Comment:
Not sure this is the right approach.
First, I would not use split on every call, but would instead cache the resulting table and reuse it.
Second, there is probaby a better way of doing this. Something probably language specific?

Reply:
this was re-written in the later commits, you can check the "Files changed" tab for a direct diff between the merge source and target.